stdlib/public/Atomics/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+#===--- CMakeLists.txt - Differentiable programming support library ------===#
```

I think this has been copy-pasted from elsewhere. Is the comment necessary?

```suggestion
#===--- CMakeLists.txt - Low-Level Atomic Operations ---------------------===#
```